content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -99,44 +99,39 @@ marked as a `metrics` file with `-M`. Its contents are:
 }
 ```
 
-> DVC doesn't force you to use any specific file names, nor does it enforce a
-> format or structure of a metrics file. It's completely user/case-defined.
-> Refer to `dvc metrics` for more details.
-
-You can view tracked metrics with DVC:
+You can view tracked metrics with `dvc metrics show `:
 
 ```dvc
 $ dvc metrics show
 Path                    avg_prec.test    avg_prec.train    roc_auc.test    roc_auc.train
 eval/live/metrics.json  0.94496          0.97723           0.96191         0.98737
 ```
 
-## Configuring plots
+[`evaluate.py`]:
+  https://github.com/iterative/example-get-started/blob/master/src/evaluate.py
+[roc-auc]:
+  https://scikit-learn.org/stable/modules/model_evaluation.html#receiver-operating-characteristic-roc
+[average precision]:
+  https://scikit-learn.org/stable/modules/model_evaluation.html#precision-recall-and-f-measures
+[metrics file]: /doc/command-reference/metrics#supported-file-formats
+
+The stage also writes [`roc_curve`] and [`confusion_matrix`] values in the
+`eval/live/plots` directory. Let's see how to visualize these better...
```

Comment:
Let's move this into the following paragraph per https://github.com/iterative/dvc.org/pull/4145#discussion_r1056031907 🙏

Reply:
Done.

Reply:
This is the one part that still doesn't read well to me. Per https://github.com/iterative/dvc.org/pull/4145#discussion_r1056031907, I thought the motivation was to mention all the plots files in this paragraph so that the next paragraph went straight to configuring them, but there is still a hanging paragraph about the feature importance image file. 

Also, I find the structure/order confusing. It starts with the roc and confusion matrix without any explanation of what those files look like, then gives a detailed explanation and example for precision-recall (seems like there must be something unique about this file for it to be listed separately with so much detail, but it's not clear why), and finally mentions a custom image after the example.

What about something like this?


> The stage also writes [plots files](/doc/user-guide/project-structure/dvcyaml-files#plots) you can visualize:
> 
> * [DVCLive](/doc/dvclive)-generated ROC curves and confusion matrices under `eval/live`.
> * Custom precision-recall curves as JSON arrays (format shown below) under `eval/prc`.
> * Custom image showing a bar chart of features' importance in `eval/importance.png`.
>
> ```
> {
>   "prc": [
>     { "precision": 0.0214, "recall": 1.0, "threshold": 0.0 },
>     ...
>     { "precision": 1.0, "recall": 0.0093, "threshold": 0.6 }
>   ]
> }
> ```

It puts everything in one paragraph with a uniform structure (💅 I also changed the JSON example back but truncated the values to focus on it being an array rather than the fields within each element of the array).

Reply:
I did try to mention all in a single paragraph as originally suggested, but it was hard to read and unclear why the code sample below only focused on one of them. I like your idea though! I applied it but put the JSON example under that specific bullet. PTAL

Reply:
One last thing: now that we have this paragraph more structured, it stick out to me that the `dvc.yaml` below it is in a different order. Should we rearrange that `dvc.yaml` to match the order of the bullets? Not a blocker, but I think it would be more readable.

Reply:
Yeah I had also noticed that it's the exact inverse order but thought that's just how the real file will look file to the user so left it like that. If we change it here would have to modify the sample repo prob? If we do we can revisit this any time @dberenbaum

Reply:
Yup looks like it's a copy/paste ATM:

https://github.com/iterative/example-get-started/blob/7e4e86ca117f1bbef288f2abebfc7c97d0a9925d/dvc.yaml#L48-L65

Reply:
Yeah, there's no functional difference, but we can reorder it in both places to match the bullets here.